packages/babel-plugin-proposal-class-properties/test/fixtures/property-tdz-decorator-interop/exec.js

```diff
@@ -0,0 +1,7 @@
+function dec() {}
```

Comment:
could we test something here? it seems that this show only the output, so technically could be just a fixture test, u dont actually evaluate the output right now so i think it doesnt justify using `exec.js`

Reply:
Before the fix `exec.js` incorrectly threw a runtime error. There was no way to determine the correct `output.js` fixture back then. Personally, I would add the (currently defunct) [`proposal-decoratoras` transform](https://github.com/babel/babel/tree/master/packages/babel-plugin-proposal-decorators) to make sure that once https://github.com/babel/babel/pull/6107 lands, independently of how both transforms interplay, they still yield valid, executable code.

Reply:
Converting to a fixture makes sense to me. I renamed to `input.js` but was unable to get an output generated locally. Committed to see what the CI does.

Also I noticed that some other tests are called `input.js` but executed? For example https://github.com/babel/babel/tree/master/packages/babel-plugin-proposal-class-properties/test/fixtures/static-property-tdz-edgest-case.